CONTRIBUTING.md

```diff
@@ -5,14 +5,20 @@
 2. Clone your fork to your machine. ```git clone https://github.com/<Your_Username>/Tusky```
 3. Create a new branch named after your change. ```git checkout -b your-change-name``` (```checkout``` switches to a branch, ```-b``` specifies that the branch is a new one)
 
+## Building
+
+Building Tusky requires Gradle 7.5 ([AGP](https://developer.android.com/studio/releases/gradle-plugin) 7.4.1) or newer. The easiest way to get this is to install [Android Studio](https://developer.android.com/studio/releases/gradle-plugin#android_gradle_plugin_and_android_studio_compatibility) Electric Eel (2022.1.1) or newer.
```

Comment:
By the way, I'm not 100% sure the gradle version here is correct, AGP 7.4.1 seems to have installed Gradle 7.6 for me. I'm just going off the values in Google's docs. It might be simpler to just give an Android Studio version, but technically some people might build without Android Studio.